packages/office-ui-fabric-react/src/components/TextField/TextField.styles.tsx

```diff
@@ -118,27 +105,27 @@ export function getStyles(props: ITextFieldStyleProps): ITextFieldStyles {
         display: 'flex',
         borderBottomWidth: 1,
         borderBottomStyle: 'solid',
-        borderBottomColor: 'inherit',
+        borderBottomColor: semanticColors.inputBorder,
         width: '100%'
       },
       hasErrorMessage && {
-        borderColor: semanticColors.errorText,
+        borderBottomColor: semanticColors.errorText,
         selectors: {
           '&:focus, &:hover': {
-            borderColor: semanticColors.errorText
+            borderBottomColor: semanticColors.errorText
           }
         }
       },
       hasErrorMessage &&
         underlined &&
```

Comment:
For the `hasErrorMessage && underlined` case, focus is still jumpy and doesn't have the high contrast border color.

Reply:
Fixed it 👍 Good catch.